packages/date-helpers/src/date-helpers.ts

```diff
@@ -0,0 +1,194 @@
+   * // Result: "2025-10-20"
+   * ```
+   *
+   * @param dateSource - The date to format, can be a Date object or a string.
+   * @returns The formatted date string.
+   */
+  public static formatIsoDate(dateSource: DateType): string {
+    // Do nothing with empty strings
+    if (dateSource === '') {
+      return dateSource;
+    }
+
+    return formatISO(getParsedDate(dateSource), {
+      representation: 'date',
+    });
+  }
+
+  /**
+   * Formats a date as a US locale date string.
+   *
+   * @example
+   * ```typescript
+   * const usDateString = DateHelpers.formatUsLocaleDate(new Date(2023, 0, 31));
+   * // Result: "01/31/2023"
+   *
+   * // or
+   *
+   * const usDateString = DateHelpers.formatUsLocaleDate("2025/10/20");
+   * // Result: "10/20/2025"
+   * ```
+   *
+   * @param dateSource - The date to format, can be a Date object or a string.
+   * @param dateFormat - Optional, a custom date format string.
+   * @returns The formatted date string.
+   */
+  public static formatUsLocaleDate(
+    dateSource: DateType,
+    dateFormat?: string,
+  ): string {
+    // Do nothing with empty strings
+    if (dateSource === '') {
+      return dateSource;
+    }
+
+    if (dateFormat) {
+      return format(getParsedDate(dateSource), dateFormat);
+    }
+
+    return format(getParsedDate(dateSource), DateHelpers.usLocaleDateFormat);
+  }
+
+  /**
+   * Formats a date as a US locale time string.
+   *
+   * @example
+   * ```typescript
+   * const usTimeString = DateHelpers.formatUsLocaleTime(new Date(2023, 0, 31, 12, 0, 0));
+   * // Result: "01/31/2023 12:00:00"
+   *
+   * // or
+   *
+   * const usTimeString = DateHelpers.formatUsLocaleTime("2025/10/20 12:00:00");
+   * // Result: "10/20/2025 12:00:00"
+   * ```
+   *
+   * @param dateSource - The date to format, can be a Date object or a string.
+   * @param dateFormat - Optional, a custom date format string.
+   * @returns The formatted time string.
+   */
+  public static formatUsLocaleTime(
+    dateSource: DateType,
+    dateFormat?: string,
+  ): string {
+    return DateHelpers.formatUsLocaleDate(
+      dateSource,
+      dateFormat ?? DateHelpers.usLocaleTimeFormat,
+    );
+  }
+
+  /**
+   * Formats a date as a string in the "Month day, year" format.
+   *
+   * @example
+   * ```typescript
+   * const monthDayYearString = DateHelpers.formatMediumDate(new Date(2023, 0, 1));
+   * // Result: "Jan 1, 2023"
+   *
+   * // or
+   *
+   * const monthDayYearString = DateHelpers.formatMediumDate("2023/01/01");
+   * // Result: "Jan 1, 2023"
+   * ```
+   *
+   * @param dateSource - The date to format, can be a Date object or a string.
+   * @returns The formatted date string.
+   *
+   * @remarks Naming Origin: Derived from the Angular DatePipe official documentation.
+   *
+   * Reference:
+   *
+   * https://angular.io/api/common/DatePipe#:~:text=mediumDate
+   */
+  public static formatMediumDate(dateSource: DateType): string {
```

Comment:
nit: Should we add `us locale` in the method name or remove it from others for consistency?